lib/chef-dk/skeletons/code_generator/files/default/repo/README.md

```diff
@@ -15,7 +13,7 @@ This repository contains several directories, and each directory contains a READ
 
 # Configuration
 
-The config file, `.chef/config.rb` is a repository specific configuration file for knife. If you're using the Chef Platform, you can download one for your organization from the management console. You can generate a new one with `knife configure`. For more information about configuring Knife, see the Knife documentation.
+The config file, `.chef/config.rb` is a repository specific configuration file for the knife command line tool. If you're using the Hosted Chef platform, you can download one for your organization from the management console. You can also generate a new config.rb by running `knife configure`. For more information about configuring Knife, see the Knife documentation.
```

Comment:
```suggestion
The config file, `.chef/config.rb` is a repository-specific configuration file for the knife command line tool. If you're using the Hosted Chef platform, you can download one for your organization from the management console. You can also generate a new config.rb by running `knife configure`. For more information about configuring Knife, see the [Knife documentation] (https://docs.chef.io/knife.html).
```

Reply:
I think the goal was to keep this light on the markdown since most people just read it in their terminal so no links in markdown

Reply:
Gotcha. Ignore the two related suggestions then! :)